.env

```diff
@@ -9,3 +9,4 @@ SITE_MAIL=admin@localhost
 ACCOUNT_MAIL=admin@localhost
 SITE_NAME='Contenta CMS Demo'
 ACCOUNT_NAME=admin
+ACCOUNT_PASS=admin
```

Comment:
Hmm, this lives in `.env.local` for the Contenta CMS installer. Any objections on moving it there?

Reply:
Well, that's exactly what I said in the corresponding issue :-)(https://github.com/contentacms/contenta_docker/issues/22#issuecomment-418733742). It was just a matter of removing work from the user.

We can move it to `.env.local`, but then we have to edit the documentation accordingly, so that the users know that they have to add the password manually, right?